YellowNotes.Api/Controllers/UserController.cs

```diff
@@ -0,0 +1,58 @@
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using YellowNotes.Core.Dtos;
+using YellowNotes.Core.Services;
+namespace YellowNotes.Api.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class UserController : ControllerBase
+    {
+        private readonly IUserService userService;
+
+        public UserController(IUserService userService) => this.userService = userService;
+
+        [HttpPost("register")]
+        public async Task<IActionResult> CreateUser([FromBody] UserDto userDto)
+        {
+            bool valid = ModelState.IsValid; 
+            if (!valid)
```

Comment:
You can just put `!ModelState.IsValid` inside the if statement. This additional variable doesn't make much sense in this case.